docs/concepts/06_messaging.md

```diff
@@ -26,15 +30,9 @@ As long as both parties keep their secret keys well protected, the combination o
 
 While encrypted, the message travels in a compact and privacy-preserving envelope format that only exposes data that the recipient requires to be able to decrypt.
 
-<!-- TODO: Replace with dynamically-generated JSON -->
-```ts
-{
-  receiverKeyUri: 'did:kilt:4qMx…66m4#0x4af3…cdcb' // indicates the key to be used when decrypting
-  senderKeyUri: 'did:kilt:4qtP…m507#0x65ac…a282' // indicates the key used to encrypt
-  ciphertext: '0xdeadbeef…' // message contents in unreadable, encrypted form
-  nonce: '0x1234…' // random data required for decryption
-}
-```
+<CodeBlock className="language-json">
+  {encryptedMessage}
```

Comment:
The cipher text seems a bit much, but I guess that's just the price we pay for proper json examples.

Reply:
Yeah indeed. I could change the object properties after it has been generated, but I think this is more indicative.